clients/mobile/App.tsx

```diff
@@ -11,6 +11,8 @@ import { AuthProvider } from './src/authentication/context/AuthContext'
 import { NotificationProvider } from './src/setup/notification/NotificationProvider'
 import * as Notifications from 'expo-notifications'
 
+// LogBox.ignoreAllLogs(true)
+
```

Comment:
We need this part of code for app build so that users don't see any unexpected system error. Do you have any other suggestion to keep it dynamic way so that it activates while building the apk?

Reply:
We can use Env.

Reply:
Let's have a look at the change I made [here](https://github.com/CraftsmenLtd/BloodConnect/pull/274/files#diff-ef0b95c9f6e89d6d03a75ed5f9061270afbcb26aa6f950e9a141112a718e1a4fR14-R18)